File: chap1/TauxQuelconque.java
// Ces programmes sont sous licence CeCILL-B V1.
class TauxQuelconque {
 
  public static void main (String [] args) {
    double ht,ttc,taux;
    System.out.println("Quel est le prix hors taxes ?");
    ht = Isn.readDouble();
    System.out.println("Quel est le taux de TVA ?");
    taux = Isn.readDouble();
    ttc = ht + ht * taux / 100;
    System.out.print("Le prix toutes taxes comprises est ");
    System.out.println(ttc);}}

File: chap1/TauxQuelconque.cpp
// Ces programmes sont sous licence CeCILL-B V1.
#include <iostream>

using namespace std;


int main()
{
   double ht,ttc,taux;
   cout << "Quel est le prix hors taxes ?" << endl;
   cin >> ht;
   cout << "Quel est le taux de TVA ?" << endl;
   cin >> taux;
   ttc = ht + ht * taux / 100;
   cout << "Le prix toutes taxes comprises est ";
   cout << ttc << endl;
}
